docs/user-guide/running-the-server.rst

```diff
@@ -118,6 +118,28 @@ You can also configure the Rasa server to fetch your model from a remote storage
 The model will be downloaded and stored in a temporary directory on your local storage system.
 For more information see :ref:`cloud-storage`.
 
+.. _server_ssl:
+
+Configuring SSL / HTTPS
+-----------------------
+
+By default the Rasa server is using HTTP for its communication. To secure the
+communication with SSL, you need to provide a valid certificate and the corresponding
+private key file.
+
+You can specify these files as part of the ``rasa run`` command:
+
+.. code-block:: bash
+
+    rasa run --ssl-certificate myssl.crt --ssl-keyfile myssl.key
+
+If you encrypted your keyfile with a password during creation, you need to add
+this password to the command:
+
+.. code-block:: bash
+
+    rasa run --ssl-certificate myssl.crt --ssl-keyfile myssl.key --ssl-password mypassword
+
```

Comment:
"The certfile string must be the path to a single file in PEM format containing the certificate as well as any number of CA certificates needed to establish the certificate’s authenticity. The keyfile string, if present, must point to a file containing the private key in. Otherwise the private key will be taken from certfile as well." 

Looks like all of this is handled here, but we might want to give more information about what goes where? Mostly wondering about `any number of CA certificates needed to establish the certificate’s authenticity` for self-signed certs

Reply:
we can add that copy if you want

the part you mentioned just means the whole cert chain needs to be present (usually nothing the user needs to worry about as far as I know - the tool creating the `.cert` file usually takes care of that)

Reply:
Ok, if the authenticity is packaged automatically by the tool i think it's fine